Sources/Exporters/OpenTelemetryProtocol/trace/utils/HTTPClient.swift

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Foundation
+
+/// Client for sending requests over HTTP.
+internal final class HTTPClient {
+    private let session: URLSession
+
+    convenience init() {
+        let configuration: URLSessionConfiguration = .ephemeral
+        // NOTE: RUMM-610 Default behaviour of `.ephemeral` session is to cache requests.
+        // To not leak requests memory (including their `.httpBody` which may be significant)
+        // we explicitly opt-out from using cache. This cannot be achieved using `.requestCachePolicy`.
+        configuration.urlCache = nil
+        // TODO: RUMM-123 Optimize `URLSessionConfiguration` for good traffic performance
+        // and move session configuration constants to `PerformancePreset`.
+        self.init(session: URLSession(configuration: configuration))
```

Comment:
@nachoBonafonte Isn't that what this does?

Reply:
Sorry, @kevinearls , i dont know what you refer to

Reply:
@nachoBonafonte This is about your comment from yesterday where you said "Yes is a good idea, anyway we should also provide a constructor which accepts an existing URLSession to use".  

I think I pointed to the wrong place though, I meant here: https://github.com/open-telemetry/opentelemetry-swift/pull/376/files#diff-2fdc8227312474239acd339b5f810132bd11efbc7c06c000a57bb24002b92a53R23